metron-platform/metron-parsing/metron-parsers-common/src/main/java/org/apache/metron/parsers/ParserRunnerImpl.java

```diff
@@ -255,7 +256,7 @@ private void initializeParsers(Supplier<ParserConfigurations> parserConfigSuppli
     if (!message.containsKey(Constants.GUID)) {
       message.put(Constants.GUID, UUID.randomUUID().toString());
     }
-    message.putIfAbsent(Fields.ORIGINAL.getName(), new String(rawMessage.getMessage()));
+    message.putIfAbsent(Fields.ORIGINAL.getName(), new String(rawMessage.getMessage(), StandardCharsets.UTF_8));
```

Comment:
This needs to care about the underlying parser's format, right? This is pulling the raw message, but that might not be `UTF_8`

Reply:
See https://github.com/justinleet/metron/pull/22